docs/error_knowledge_base.md

```diff
@@ -418,14 +419,36 @@ See also: [Submitting a Package](how_to_add_packages.md#submitting-a-package).
 The recipe missess [short_paths](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#short-paths) attribute.
 It may happen due to the very long paths within source, build or package directories during the package creating.
 Consider adding the following code:
+
 ```python
 class SomeRecipe(ConanFile):
     ...
 
     short_paths = True
 ```
 
-See also: (Maximum Path Length Limitation)[https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd].
+See also: [Maximum Path Length Limitation](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=cmd).
+
+#### **<a name="KB-H068">#KB-H068</a>: "TEST_TYPE MANAGEMENT"**
+
+https://github.com/conan-io/hooks/commit/ad867ad0a3656fe77ecc964e9b8f3657995f0d5d
```

Comment:
we need to take a time, and write some actual descriptions here and below

Reply:
Thats why I left this as a draft 🙃 there was no links to follow and I am not sure where the rules came from.

Reply:
**KB-H068**: Conan v2 requires `self.test_reference_str` to be explicit in the recipe. Using `test_type = "explicit"` prevents Conan v1 from injecting the tested package automagically as a requires, so it forces to use `self.test_reference_str` explicitly.

Reply:
```suggestion
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#changes-in-the-test-package-recipe
In Conan 2.0, the `test_package/conanfile.py` needs to declare the requirement being tested explicitly. To be prepared you have to set the attribute `test_type="explicit"` (this will be ignored in 2.0) to make Conan activate the explicit mode, then declaring the requirement using the `self.tested_reference_str` that contains the reference being tested.
```